libs/cardano-ledger-core/src/Cardano/Ledger/Plutus/CostModels.hs

```diff
@@ -37,7 +37,12 @@ module Cardano.Ledger.Plutus.CostModels (
   costModelToMap,
   costModelFromMap,
   decodeCostModelFailHard,
-  CostModels (..),
+  CostModels,
```

Comment:
Maybe this will be easier to find if we move it up after `CostModel` export ?

Reply:
It'll actually be better if I'll organize functions and type relative to `CostModel` vs `CostModels`. So'll create a section for `CostModels` and will move this export to the top of that section.